packages/components/filters/src/filters.visualroute.jsx

```diff
@@ -1,11 +1,64 @@
 import { Suite, Spec } from '../../../../test/percy';
+import { FiltersWithState } from './fixtures/filters-with-state';
 
 export const routePath = '/filters';
 
+const selectedPrimaryColors = [
+  'Test label that is very very long for testing',
+  'green',
+  'yellow',
+  'orange',
+  'red',
+];
+
+const selectedSecondaryColors = ['purple', 'forest'];
+
 export const component = () => (
   <Suite>
-    <Spec label="this is a placeholder test">
-      {/**ACTUAL TEST GOES HERE */}
+    <Spec label="Filters - closed with 0 filters selected">
+      <FiltersWithState />
+    </Spec>
+    <Spec label="opened with 0 filters selected">
+      <FiltersWithState defaultOpen={true} />
+    </Spec>
+    <Spec label="closed with 1 filter selected">
+      <FiltersWithState selectedSecondaryColors={selectedSecondaryColors} />
```

Comment:
IDK why this test is showing that it pops a menu in the percy snapshots - it doesnt when I run the percy app or storybook locally, or in the test where 2 filters with data are mounted when the filter is closed.  This is especially frustrating because percy disables animation, so there is no way to open the filter menu dropdown with puppeteer anyway.  I am inclined to call this percy's fault.